tests/test_utils/tests.py

```diff
@@ -10,7 +10,7 @@
 from django.core.files.storage import default_storage
 from django.core.urlresolvers import NoReverseMatch, reverse
 from django.db import connection, router
-from django.forms import EmailField, IntegerField
+from django.forms import CharField, EmailField, Form, IntegerField
```

Comment:
@timgraham I just added more individual imports here rather than causing (and requiring) a "refactoring" PR or commit as you had suggested. That doesn't cause touching of previously-existing code. Wrt all other changes, all improvements you suggested were applied. Thanks again.